### PR TITLE
Catch and handle errors in geoJson data fetching gracefully

### PR DESCRIPTION
**Pull request by vesameskanen**

### CODE REVIEW

File: app/store/GeoJsonStore.js [hunk 1]
```diff
@@ -104,23 +107,28 @@ class GeoJsonStore extends Store {
...
+        const responses = await Promise.all(urlArr.map(u => getJson(u)));
+        const mapped = responses.map(r => {
+          if (metadata) {
+            MapJSON(r, metadata);
+          }
+          return styleFeatures(r);
+        });
+        for (let i = 1; i < mapped.length; i++) {
+          mapped[0].features = mapped[0].features.concat(mapped[i].features);
         }
```
Comment (partisaani) [inline app/store/GeoJsonStore.js]: Could these loops be combined with something like
```javascript
  const features = [];
  const mapped = responses.map(r => {
    if (metadata) {
      MapJSON(r, metadata);
    }
    const styledFeature = styleFeatures(r);
    features.push(...styledFeature.features);
    return styledFeature;
  });

  mapped[0].features = features;
```

**Review by partisaani — APPROVED** [review]
LGTM with suggestion

File: app/store/GeoJsonStore.js (hunk 1 above)
Comment (vesameskanen) [reply app/store/GeoJsonStore.js]: ok, I will refactor

Comment (vesameskanen) [reply app/store/GeoJsonStore.js]: logic is now much easier to understand

Comment (partisaani) [reply app/store/GeoJsonStore.js]: Even better! 👏

**Review by partisaani — APPROVED** [review]

**Pull request merged**